packages/esm-ward-app/src/config-schema-admission-request-note.ts

```diff
@@ -0,0 +1,9 @@
+import { Type } from '@openmrs/esm-framework';
+
+export const admissionRequestNoteRowConfigSchema = {
+  conceptUuid: {
+    _type: Type.UUID,
+    _description: 'Required. Identifies the concept for the admission request note.',
+    _default: '161011AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
```

Comment:
No sure what our best practices are for defaults, but I'm thinking we should only include a default if it's something in CIEL and included in the ref app.  (which may be true for the above concept).

Reply:
Yeah, the bar for putting a default UUID seems to be whether it's included in the ref app. 

In this case, the UUID is both in CIEL and in the ref app.